public/js/combinationalAnalysis.js

```diff
@@ -32,8 +34,11 @@ createCombinationalAnalysisPrompt=function(scope=globalScope){
 }
 function createBooleanPrompt(inputListNames,outputListNames,scope=globalScope){
 
-    inputListNames=inputListNames||(prompt("Enter inputs separated by space").split(' '));
-    outputListNames=outputListNames||(prompt("Enter outputs separated by space").split(' '));
+    inputListNames=inputListNames||(prompt("Enter inputs separated by commas").split(','));
+    outputListNames=outputListNames||(prompt("Enter outputs separated by commas").split(','));
+    outputListNamesInteger=[];
+    for (var i = 0; i < outputListNames.length; i++)
+        outputListNamesInteger[i] = 7*i + 13;//assigning an integer to the value, 7*i + 13 is random
```

Comment:
Why use something like this why can it simply be the index i itself?

Reply:
Idk I just felt that the index i might just repeat somewhere. No good reason. Just a random feeling, so I put in this random expression

Reply:
Okay